client/src/modules/cohort/index.jsx

```diff
@@ -198,58 +198,55 @@ class Cohort extends PureComponent {
           <div className="wrapper">
             <div className="cohort">
               <CohortHeader details={cohortDetails} />
-              {pendingForDetails ? (
-                <Preloader />
-              ) : (
-                <Fragment>
-                  <MembersBox
-                    isCurrentUserAnOwner={this.checkIfOwner()}
-                    members={members}
-                    route={Routes.COHORT}
-                  />
-                  <GridList
-                    color={CardColorType.ORANGE}
-                    icon={<ListIcon />}
-                    items={lists}
-                    name="Lists"
-                    onAddNew={this.handleDialogContext(DialogContext.CREATE)}
-                    pending={pendingForDetails}
-                    placeholder={`There are no lists in the ${name} cohort!`}
-                    route={Routes.LIST}
-                  />
-                  {!isArchived && this.checkIfOwner() && (
-                    <button
-                      className="link-button"
-                      onClick={this.handleDialogContext(DialogContext.ARCHIVE)}
-                      type="button"
-                    >
-                      {`Archive the "${name}" cohort`}
-                    </button>
+              <div className="cohort__details">
+                <MembersBox
+                  isCurrentUserAnOwner={this.checkIfOwner()}
+                  members={members}
+                  route={Routes.COHORT}
+                />
+                <GridList
+                  color={CardColorType.ORANGE}
+                  icon={<ListIcon />}
+                  items={lists}
+                  name="Lists"
+                  onAddNew={this.handleDialogContext(DialogContext.CREATE)}
+                  placeholder={`There are no lists in the ${name} cohort!`}
+                  route={Routes.LIST}
+                />
+                {!isArchived && this.checkIfOwner() && (
```

Comment:
wouldn't be better to send isOwner prop for cohort for currentUser, as we do for lists??